source/sass/type.scss

```diff
@@ -48,58 +49,58 @@ a {
   font-weight: 100;
   letter-spacing: 0;
 
-  @include scaleText(36px);
+  @include scaleText(36px, 48px);
   @include black-box-white-text;
 }
 
 .h2-headings-white {
   letter-spacing: 1px;
   text-transform: uppercase;
 
-  @include scaleText(18px);
+  @include scaleText(16px, 24px);
   @include black-box-white-text;
 }
 
 .h2-typeaccents-gray {
   font-family: Arvo;
   font-weight: 400;
   color: #ccc;
-  line-height: 80px;
+  line-height: 1.5;
 
-  @include scaleText(54px);
+  @include scaleText(56px, 72px);
 }
 
 .h3-black {
   font-family: Arvo;
   font-weight: 100;
   letter-spacing: 0;
-  line-height: 48px;
+  line-height: 1.7;
```

Comment:
All the sizes have been tweaked, including the larger sizes, so I think design will have to take a look at the build and give me specific values.